src/sentry/static/sentry/app/components/badge.jsx

```diff
@@ -4,7 +4,7 @@ import {cx} from 'react-emotion';
 
 class Badge extends React.Component {
   static propTypes = {
-    text: PropTypes.string,
+    text: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
Would `PropTypes.node` be better (includes string, number, and a react component)?